### PR TITLE
Sprockets 3.x support

### PR DESCRIPTION
**Pull request by pkrefta**
Hello :smile: 

I've started working on adopting Sprockets 3.x line and everything seems to be working as expected although I still want to make some tests.

Using both 2.x and 3.x requires version conditionals and I'm suggesting dropping 2.x support since there aren't any real disadvantages.

What do you think about that @petebrowne ?

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -28,7 +28,20 @@ def env
   # Returns a fresh context, that can be used to test helpers.
   def context(logical_path = 'application.js', pathname = nil)
     pathname ||= Pathname.new(File.join('assets', logical_path)).expand_path
-    env.context_class.new env, logical_path, pathname
+
+      # env.context_class.class_eval do
```
Comment (petebrowne) [inline spec/spec_helper.rb]: Remove this commented out area and this looks pretty good to go

**Comment by petebrowne** [comment]
Are the version conditionals in the specs or the lib itself? I'd like to maintain compatibility if possible, so I don't have to patch multiple versions if there's a bug

**Comment by pkrefta** [comment]
@petebrowne Conditionals are both in specs & lib but I think there are not too intrusive. I've added support Sprockets 2.x but from 2.2 and up because lower versions have some hickups.

**Comment by petebrowne** [comment]
Thanks for the help with this! I'd love some help with updating sprockets-sass if you're interested.

**Comment by pkrefta** [comment]
@petebrowne I'll see what I can do with **sprockets-sass** :) Waiting for release.

**Comment by petebrowne** [comment]
Released: https://rubygems.org/gems/sprockets-helpers/versions/1.2.0

**Comment by brandonweiss** [comment]
@pkrefta Thanks!

**Pull request merged**